saskatoon/unittests/sitebase/test_utils.py

```diff
@@ -0,0 +1,51 @@
+from datetime import datetime
+from django.utils import timezone
+import pytest
+
+from sitebase.utils import parse_naive_datetime, rgetattr
+
+
+@pytest.mark.parametrize("unparsed", ["2025-12-18+16:00", "2025-12-18+18:00", "2025-12-19+15:00"])
+def test_parse_naive_datetime(unparsed):
+    tzinfo = timezone.get_current_timezone()
+    parsed = parse_naive_datetime(unparsed)
+
+    # datetime.now(tz=tzinfo) is sometimes different from datetime.replace(tzinfo=tzinfo)
+    assert parsed.tzinfo == datetime.now(tz=tzinfo).tzinfo
```

Comment:
Could we test more than the timezone part? 

I think it'd be worth asserting equality between 2 naive datetimes